ci-operator/jobs/janus-idp/backstage-showcase/janus-idp-backstage-showcase-main-presubmits.yaml

```diff
@@ -13,7 +13,6 @@ presubmits:
     labels:
       ci.openshift.io/generator: prowgen
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
-    max_concurrency: 1
```

Comment:
`max_concurrency: 1` This is for PR check to run serially. lets keep it. you need to manually update this file to add it.

Reply:
Thanks for the catch. I didn't intend to change it.